### PR TITLE
Retry on get connection

### PR DESCRIPTION
**Pull request by lferran**

### CODE REVIEW

**Comment by lferran** [comment]
tests are passing locally
image

File: guillotina_amqp/amqp.py [hunk 1]
```diff
@@ -68,7 +74,13 @@
     if name in connections:
         connection = connections[name]
         return connection["channel"], connection["transport"], connection["protocol"]
-    channel, transport, protocol = await connect()
+    try:
+        channel, transport, protocol = await connect()
```
Comment (vangheem) [inline guillotina_amqp/amqp.py]: `connect()` is only called at start so if this error happens, there will not be anything in the connections cache to remove actually right?

Maybe we also just need `@backoff` here and attempt to retry on this exception?

This is my bad. I thought this error was manifesting somewhere else.

Comment (lferran) [reply guillotina_amqp/amqp.py]: :+1:

**Comment by vangheem** [comment]
5.0.12 released

**Pull request merged**